src/platform/unix.rs

```diff
@@ -188,9 +206,15 @@ pub trait EventLoopWindowTargetExtUnix {
   // ///
   // /// The pointer will become invalid when the winit `EventLoop` is destroyed.
   // fn wayland_display(&self) -> Option<*mut raw::c_void>;
+
+  /// Returns the linux-specific window target struct
+  fn window_target(&self) -> &UnixEventLoopWindowTarget<T>;
```

Comment:
this shouldn't be exposed, and where ever this is needed, should be replaced with the user facing type `EventLoopWindowTarget` not the platform specific one

Reply:
Ah okay, I think I figured out what you're getting at.  Removed this method and I think everything works now.